integration-cli/docker_cli_events_unix_test.go

```diff
@@ -396,35 +396,28 @@ func (s *DockerDaemonSuite) TestDaemonEvents(c *testing.T) {
 	defer os.Remove(configFilePath)
 
 	daemonConfig := `{"labels":["foo=bar"]}`
-	fmt.Fprintf(configFile, "%s", daemonConfig)
-	configFile.Close()
+	_, err = fmt.Fprintf(configFile, "%s", daemonConfig)
+	assert.NilError(c, configFile.Close())
+	assert.NilError(c, err)
 	s.d.Start(c, fmt.Sprintf("--config-file=%s", configFilePath))
 
-	// Get daemon ID
-	out, err := s.d.Cmd("info")
+	daemonID, err := s.d.Cmd("info", "--format", "{.ID}")
```

Comment:
`{{.ID}}`

Reply:
You could make this `{{.ID}}|{{.Name}}` and then split it on `|` to only do one call to info.

Reply:
LOL I had that initially

Reply:
I still think that would be better, but let me know if you disagree